eq-author/src/App/page/PropertiesPanel/GroupedAnswerProperties/index.test.js

```diff
@@ -479,11 +479,13 @@ describe("Grouped Answer Properties", () => {
 
       expect(screen.getByTestId("MaxCharacterTooBig")).toBeVisible();
       expect(screen.getByTestId("MaxCharacterTooBig").textContent).toBe(
-        characterErrors.CHAR_LIMIT_2000_EXCEEDED
+        "Error:" +
+          String.fromCharCode(160) +
+          characterErrors.CHAR_LIMIT_2000_EXCEEDED
```

Comment:
```suggestion
        {`Error: ${characterErrors.CHAR_LIMIT_2000_EXCEEDE}`}
```

ES6 Syntax. Omitted the String.fromCharCode(160) as that returns a whitespace, which we can put into the string.